prodtests/full-system-test/aggregator-workflow.sh

```diff
@@ -221,11 +221,14 @@ fi
 lanesFactorize=6
 nTFs=1000
 nTFs_SAC=1000
+IDC_DELTA="--disable-IDCDelta true" # by default, deltas are off
```

Comment:
I have to say I don't understand this ENABLE_IDC_DELTA=empty string. That is a bit confusing to use. The default in the code is to have --disable-IDCDelta disabled. If I understand correctly, we want to enable it by default for pp, or upon a special setting. I'd simply do:
```
IDC_DELTA=""
[[ $BEAMTYPE != "PbPb" && "0$ENABLE_IDC_DELTA" != "01" ]] && IDC_DELTA="--disable-IDCDelta true"
```
That is more intuitive to me.

Reply:
This was my first way. Then I thought that if at P2 they just write `ENABLE_IDC_DELTA=true` or `ENABLE_IDC_DELTA=` or `ENABLE_IDC_DELTA=1`, which are all quite clear to me, they should all set it.  
If you think that only the `1` should do it, then I can f course change it. 
I don't have a strong preference, I have to say, but since you do, I will modify it.

Reply:
BTW, the idea is to enable it by default in PbPb, I think this was clear (I point it out since you wrote `we want to enable it by default for pp`).

Reply:
ok..., that was a typo indeed. Fixed it now. But the code I posted was correct :)

Reply:
Please check the new commit. It is different from what you proposed.